### PR TITLE
Bau tidy up

### PR DESCRIPTION
**Pull request by andrew-moores**
## What

Tidy up following comments on PR #97.

Changed name of obfuscate function to better reflect what we are actually doing - truncate.
Moved the test public key for IPV to the new shared keys file.

## How to review

1. Code Review

### CODE REVIEW

File: ipv-stub/src/endpoints/ipv-token.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ function parseBody(body: string): Result<Partial<ValidatedParams>> {
 
   logger.info("Handling token request with parameters:");
   for (const [key, value] of Object.entries(validParameters)) {
-    const loggedValue = shouldObfuscate(key) ? obfuscate(value) : value;
+    const loggedValue = shouldObfuscate(key) ? truncate(value) : value;
```
Comment (BeckaL) [inline ipv-stub/src/endpoints/ipv-token.ts:63]: Nice, should we also change the `shouldObfuscate` function name? (I can see an argument for it either way so not too fussed)

**Review by BeckaL — COMMENTED** [review]
Minor comment, LGTM

File: ipv-stub/src/endpoints/ipv-token.ts (hunk 1 above)
Comment (alhcomer) [reply ipv-stub/src/endpoints/ipv-token.ts:63]: fwiw I have a slight preference for shouldTruncate but I'm also not fussed

Comment (andrew-moores) [reply ipv-stub/src/endpoints/ipv-token.ts:63]: My thinking was we decide if we should obfuscate and then decide how, in this case just simply truncating.  If we get any more complex requirements we can change it then.

**Review by BeckaL — APPROVED** [review]

**Pull request merged**